### ISSUE TITLE
Builds failing 

### ISSUE THREAD

**Issue by mehmandarov** [sergiokopplin/indigo#304]
All the builds on GitHub are failing with this message. We should fix that:

```
Gem::RuntimeRequirementNotMetError: ruby_dep requires Ruby version >= 2.2.5, ~>
2.2. The current ruby version is 2.2.0.
An error occurred while installing ruby_dep (1.5.0), and Bundler
cannot continue.
Make sure that `gem install ruby_dep -v '1.5.0'` succeeds before bundling.
In Gemfile:
  github-pages was resolved to 182, which depends on
    jekyll-avatar was resolved to 0.5.0, which depends on
      jekyll was resolved to 3.7.3, which depends on
        jekyll-watch was resolved to 2.0.0, which depends on
          listen was resolved to 3.1.5, which depends on
            ruby_dep
The command "eval bundle install --jobs=3 --retry=3 " failed. Retrying, 2 of 3.
```

**Comment by drmenguin** [comment]
Change `rvm` in `travis.yml` to `2.2.5`.

**Comment by mehmandarov** [comment]
Can you submit a PR, @drmenguin?

**Comment by drmenguin** [comment]
@mehmandarov I'm not sure how to submit pull requests on github. I opened another issue pointing attention to where the change should be made.

**Comment by mehmandarov** [comment]
I have verified your fix. It works. Thank you very much. I will update the code.

**Comment by mehmandarov** [comment]
Thanks, @drmenguin!

**Comment by mehmandarov** [comment]
I have implemented the changes and sent a PR: #307.
